plugins/community/mercari-grpc-federation/v0.9.2/buf.plugin.yaml

```diff
@@ -0,0 +1,10 @@
+version: v1
+name: buf.build/community/mercari-grpc-federation
+plugin_version: v0.9.2
+source_url: https://github.com/mercari/grpc-federation
+integration_guide_url: https://github.com/mercari/grpc-federation/blob/main/docs/getting_started.md
+description: Generates a gRPC server by writing a custom option in Protocol Buffers
```

Comment:
I know this is from the GitHub description, but is there a better one we could use to better describe what this plugin does?

Reply:
```suggestion
description: gRPC Federation automatically generates a BFF (Backend for frontend) server that aggregates and returns the results of gRPC protocol-based microservices.
```